### ISSUE TITLE
Support for VS2019

### ISSUE THREAD

**Issue by CZEMacLeod** [thomaslevesque/AutoRunCustomTool#22]
It appears to be available but does not install on VS2019

```
11/01/2019 16:32:44 - 		-------------------------------------------------------
11/01/2019 16:32:44 - 		Identifier   : Microsoft.VisualStudio.Component.CoreEditor
11/01/2019 16:32:44 - 		Name         : Visual Studio core editor
11/01/2019 16:32:44 - 		Version      : [15.0.26004.1,16.0)
11/01/2019 16:32:44 - 		Error        : The prerequisite version specified does not match the version installed
```

**Comment by thomaslevesque** [comment]
hi @CZEMacLeod, thanks for reporting this, I'll look into it.

**Comment by CZEMacLeod** [comment]
@thomaslevesque Any progress on this? I find this tool very useful, and miss having it in VS2019.
Same error on the VS2019 RC BTW.

**Comment by CZEMacLeod** [comment]
@thomaslevesque I found this article on updating to support 2019 [How to upgrade extensions to support Visual Studio 2019](https://madskristensen.net/blog/how-to-upgrade-extensions-to-support-visual-studio-2019/)
I think the change is just to change to this
```
<Prerequisites> 
    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,)" DisplayName="Visual Studio core editor" /> 
</Prerequisites>
```

**Comment by thomaslevesque** [comment]
@CZEMacLeod, sorry, no progress yet.

> I found this article on updating to support 2019 [How to upgrade extensions to support Visual Studio 2019](https://madskristensen.net/blog/how-to-upgrade-extensions-to-support-visual-studio-2019/)

Yeah, I know, but currently I'm unable to build the current version, so I'll have to change a few other things.

**Comment by thomaslevesque** [comment]
OK, it was a bit easier than I feared. I published a 2.2.0 version that supports VS2019.
You'll see a warning because the extension is still using a deprecated API (synchronously loaded package). I'll fix it later, but I'll have to drop support for versions of VS older than VS2015.

**Comment by blodgettb** [comment]
I got version 3 to work by: get the source code, "Remove and Sort Usings", rebuild all, quit VS2019, and run AutoRunCustomTool\bin\Debug\AutoRunCustomTool.vsix which was just built.
